samples/performance/image_compression_control/README.adoc

```diff
@@ -0,0 +1,291 @@
+
+=== Fixed-rate compression [[fixed_rate_compression]]
+
+On the other hand, a compression scheme may use a constant bitrate for all blocks, and in this case the compression is defined as fixed-rate.
+This means that in some cases a block will lose some information, and thus the compression is lossy.
+Therefore the device may not enable it automatically, and the developer must opt-in using the Vulkan image compression control extensions.
+
+Recent devices with Arm GPUs support Arm Fixed Rate Compression (AFRC), which achieve high quality results even with the highest compression ratios.
+For instance, see below for images saved from a Pixel 8 device:
+
+[.center, cols="a,a,a"]
+|===
+| Default compression
+| 2BPC Fixed-rate compression
+| Pixel difference
+| image::./images/default.png[Default compression, 400]
+| image::./images/fixed_rate_2BPC.png[Fixed-rate compression, 400]
+| image::./images/compare.png[Pixel difference, 400]
+|===
+[.text-center]
+__Space Module scene compression comparison__
+
+Since the difference is not noticeable with the naked eye, this is sometimes referred to as "visually lossless" compression.
+Software like https://imagemagick.org/script/compare.php[imagemagick allows to compare] the images and obtain a https://en.wikipedia.org/wiki/Peak_signal-to-noise_ratio[PSNR] quality estimation, in this case with the high value of 49.8dB:
+
+[,shell]
+----
+$ magick compare -metric PSNR default.png fixed_rate_2BPC.png compare.png
+49.8487 (0.498487)
+----
+
+There are some performance benefits associated with fixed-rate compression, as described below.
+
+=== Memory footprint savings
+
+Images compressed with a fixed-rate will always consume less memory.
+In this case, an image compressed with a 2BPC bitrate results in a 65% reduction compared to uncompressed. 
+
+image::./images/footprint.png[Memory footprint savings, 700, align="center"]
+
+In this case, the slightly larger size of images compressed with AFBC is expected, as variable bitrates require enough space for the worse case (uncompressed) as well as some extra storage for compression-related metadata.
+
+=== Bandwidth savings
+
+The sample allows to observe an estimate of bytes being written out to main memory.
+On this device the write bandwidth difference between uncompressed and fixed-rate compression is approximately 38%: 
+
+image::./images/bandwidth.png[Bandwidth savings, 700, align="center"]
+
+Bandwidth savings coming from image compression depend on the pixels being compressed.
+Moving the camera and showing different distribution of colors in the frame changes the results.
+Be sure to profile your application and verify which compression scheme is optimal in each case.
+For instance, images with a high proportion of solid color (e.g. normals or material properties) may be more optimally compressed with variable bitrates than with fixed-rate.
+This is the case for the Space Module scene shown above.
+
+== VK_KHR_image_compression_control
```

Comment:
Shouldn't this be `VK_EXT_image_compression_control`? There is no extension named `VK_KHR_image_compression_control`.